src/coreclr/inc/executableallocator.h

```diff
@@ -0,0 +1,73 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+//
+
+//
+// Allocator and holders for double mapped executable memory
+//
+
+#pragma once
+
+#include "utilcode.h"
+#include "ex.h"
+#include <intrin.h>
```

Comment:
Does this file really need intrin.h ?

Reply:
It doesn't, it is a remainder of other changes I have in the branch with the complete set of W^X changes. But it is used only for logging stuff that will go away.